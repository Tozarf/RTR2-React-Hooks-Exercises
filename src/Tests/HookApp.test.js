import { shallow } from 'enzyme'
import React from 'react'
import HookApp from "../HookApp"

describe('testing <HookApp/>', () => {

    test('should match snapshot', () => {
        
        const wrapper= shallow(<HookApp/>)

        expect(wrapper).toMatchSnapshot()

    })
    
    
})
