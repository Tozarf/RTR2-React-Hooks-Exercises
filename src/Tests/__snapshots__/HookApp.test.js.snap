// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing <HookApp/> should match snapshot 1`] = `
<div>
  <h1>
    hola mundo 
  </h1>
</div>
`;
